### ISSUE TITLE
[BUG] Misaligned symbols on compact mode calendar

### ISSUE THREAD

**Issue by ZeroXNoxus** [vigoren/foundryvtt-simple-calendar#580]
**Describe the bug**
The icons displaying moon phases, seasons and close/popout are misaligned in the compact view:
![image](https://github.com/vigoren/foundryvtt-simple-calendar/assets/29397572/d3a55e61-a2af-4253-b218-1077dc149e69)

**To Reproduce**
1. Configure a calendar
2. Double click on calendar to minimze
3. Issue is reproduced

**Expected behavior**

**Screenshots**

**Foundry Information**
 - Foundry Version: 11.315
 - Game System: DnD5e v2.1.4
 - Simple Calendar Version: v2.4.3

**Browser (please complete the following information):**
 - Browser Chrome (current)

**Comment by vigoren** [comment]
Does this happen if Simple Calendar is the only module active?

I have not been able to reproduce the icons becoming misaligned

**Comment by ZeroXNoxus** [comment]
I have analyzed in my instance and I have found out that a certain module adds a pop-out button which can be toggled off to the calendar and sets the styling to "display: block;" which messes with the alignment in Simple Calendar.

Aditionally another module (mine) added a filter to the SC window, which caused the icon to slide out of position. Sorry for the wrong alert here, I have fixed this issue in my module and this should  solve the issue.
